test/e2e/set/scyllacluster/scyllamanager.go

```diff
@@ -78,8 +76,7 @@ var _ = g.Describe("Scylla Manager integration", func() {
 		sc, err = f.ScyllaClient().ScyllaV1().ScyllaClusters(f.Namespace()).Patch(ctx, sc.Name, types.MergePatchType, patchData, metav1.PatchOptions{})
 		o.Expect(err).NotTo(o.HaveOccurred())
 		o.Expect(sc.Spec.Repairs).To(o.HaveLen(1))
-		o.Expect(sc.Spec.Repairs[0].Name).To(o.Equal("weekly"))
-		o.Expect(sc.Spec.Repairs[0].Interval).To(o.Equal(pointer.Ptr("7d")))
```

Comment:
is this a flaky part? normally we'd keep tests until the deprecation period ends / while the field is still usable

Reply:
It's not flaky and we test it in our unit tests - I didn't see the point of keeping them here

Reply:
Deprecation means the field usage is discouraged but it can't stop working the other day. There needs to be a graceful time for migration so I still see the value in keeping the e2e test for it unless it brings us some discomfort.

Reply:
It doesn't, but the e2e doesn't really "test" it - meaning we're not waiting for the next start of the task either way, it just verifies that the field was propagated to the manager state. The same functionality is tested with the other fields in the test spec. So it doesn't really matter altogether.

Reply:
ok